packages/migrations/src/ensureDockerNetworkConfig/getDockerNetworkNameFromSubnet.ts

```diff
@@ -0,0 +1,23 @@
+import { docker } from "@dappnode/dockerapi";
+
+/**
+ * Get a docker network name from a docker subnetif any, otherwise undefined
+ *
+ * @param networkSubnet "172.33.0.0/16"
+ *
+ * @returns "dncore_network" || undefined
+ */
+export async function getDockerNetworkNameFromSubnet(
```

Comment:
This is bit more complicated than it seems. This function needs to return docker network even when there is a partial match in subnet space. For example, if we pass as an argument `networkSubnet = 10.20.0.192/26`, it needs to return network which has `10.20.0.0/24` specified as first range overlaps with the second one.

Reply:
good to know, @dappnodedev can you take this into account